src/test/util.py

```diff
@@ -690,6 +690,7 @@ class wally_psbt(Structure):
     ('wally_wif_to_bytes', c_int, [c_char_p, c_uint32, c_uint32, c_void_p, c_size_t]),
     ('wally_wif_to_public_key', c_int, [c_char_p, c_uint32, c_void_p, c_size_t, c_size_t_p]),
     ('wally_witness_multisig_from_bytes', c_int, [c_void_p, c_size_t, c_void_p, c_size_t, POINTER(c_uint32), c_size_t, c_uint32, POINTER(POINTER(wally_tx_witness_stack))]),
+    ('wally_witness_p2tr_from_sig', c_int, [c_void_p, c_size_t, POINTER(POINTER(wally_tx_witness_stack))]),
```

Comment:
Java isn't fully automated yet, for new funcs you need to add an entry to `src/swig_java/swig.i`. For this PR that would look like:
```
diff --git a/src/swig_java/swig.i b/src/swig_java/swig.i
index 580dee20..32c45dc7 100644
--- a/src/swig_java/swig.i
+++ b/src/swig_java/swig.i
@@ -918,6 +918,7 @@ static jobjectArray create_jstringArray(JNIEnv *jenv, char **p, size_t len) {
 %returns_size_t(wally_scriptsig_p2pkh_from_sig);
 %returns_size_t(wally_scriptsig_p2pkh_from_der);
 %returns_size_t(wally_scriptsig_multisig_from_bytes);
+%returns_struct(wally_witness_p2tr_from_sig, wally_tx_witness_stack);
 %returns_struct(wally_witness_p2wpkh_from_sig, wally_tx_witness_stack);
 %returns_struct(wally_witness_p2wpkh_from_der, wally_tx_witness_stack);
 %returns_struct(wally_witness_multisig_from_bytes, wally_tx_witness_stack);
```